tests/migration_test.go

```diff
@@ -3375,7 +3340,37 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			})
 
 			Context("[Serial] with node tainted during node drain", Serial, func() {
+
+				var controlPlaneNodeSetUnschedulable string
+
+				resetNodeSchedulable := func() {
+					libnode.SetNodeSchedulable(controlPlaneNodeSetUnschedulable, virtClient)
+
+				}
+
+				expectVMIMigratedToAnotherNode := func(vmiName, sourceNodeName string) {
+					EventuallyWithOffset(0, func() error {
```

Comment:
```suggestion
					EventuallyWithOffset(1, func() error {
```
? Otherwise it's just an `Eventually()`

Reply:
Thanks so much! Fixed